### DISCUSSION TITLE
Support for local folders

### DISCUSSION THREAD

**Discussion by Sim7300** [elementary/mail#1006] [Ideas]
I really like the look of the Mail app, but its sadly not useable for me in the Moment, because I have to work with local Mail folders very much. If there are more people using them, would the support be possible? In the moment i am using evolution instead so no problem ;)

**Comment by teamcons** [comment]
Hello
You can file a Feature Request in the Issues tab, by clicking on "New Issue". On the form you can describe it and explain what need it answers
I alas cannot do it for you, i cant really defend the case of a local folder because i dont use that.

"Discussions" is less often looked through, it is less part of the workflow, and more for helping casual users, or longer term discussions
